lldb/test/Shell/Commands/command-breakpoint-col.test

```diff
@@ -3,8 +3,10 @@
 # RUN: %clang_host -g -O0 %S/Inputs/main.c -o %t.out
 # RUN: %lldb -b -o 'help breakpoint set' -o 'breakpoint set -f main.c -l 2 -u 21' %t.out | FileCheck %s --check-prefix HELP --check-prefix CHECK
 # RUN: %lldb -b -o 'help _regexp-break' -o 'b main.c:2:21' %t.out | FileCheck %s --check-prefix HELP-REGEX --check-prefix CHECK
+# RUN: not %lldb -b -o 'breakpoint foo' %t.out -o exit 2>&1 | FileCheck %s --check-prefix ERROR-MSG
```

Comment:
Putting this into a breakpoint column test (I think that's what "col" stands for) feels rather ad hoc. If there's nothing better, then just create a new test.

You should also check (add a test) that the output makes sense for nested multiword commands (e.g. "watchpoint set foo").

Reply:
done